hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/fees/CongestionPricingTest.java

```diff
@@ -169,7 +154,7 @@ final Stream<DynamicTest> canUpdateMultipliersDynamically2() {
                         "1,7x",
                         "fees.minCongestionPeriod",
                         tmpMinCongestionPeriod),
-                fileUpdate(THROTTLE_DEFS).payingWith(EXCHANGE_RATE_CONTROL).contents(artificialLimits.toByteArray()),
+                overridingThrottles("testSystemFiles/artificial-limits-congestion.json"),
```

Comment:
yup, indeed this test looks better now, and also you can't really make mistakes in the override/revert